app/src/main/java/com/globant/practice/Presenter/SplashPresenter.java

```diff
@@ -0,0 +1,46 @@
+package com.globant.practice.Presenter;
+
+import android.util.Log;
+
+import com.globant.practice.View.Activities.SplashActivity;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+import javax.inject.Inject;
+
+/**
+ * Created by jonathan.vargas on 30/03/2017.
+ */
+
+public class SplashPresenter {
+    SplashActivity splash;
```

Comment:
The presenter MUST NOT know of Android framework. So it should not handle activities, context, resources or any other Android component